neuralmonkey/decoders/decoder.py

```diff
@@ -43,8 +43,8 @@ def __init__(self,
                      tf.Tensor]]=None,
                  use_attention: bool=False,
                  embeddings_encoder: Optional[Any]=None,
-                 rnn_cell: str='GRU',
                  attention_on_input: bool=True,
+                 rnn_cell: str='GRU',
```

Comment:
Tohle se tam nakopírovalo dvakrát při rebasu a byl jsem línej to pak rozebírat nějakym interkativním rebasem.